src/lib/eZ/RichText/Converter/Render/Template.php

```diff
@@ -115,6 +117,21 @@ protected function processTemplate(DOMDocument $document, DOMXPath $xpath, DOMEl
         }
     }
 
+    /**
+     * Provides attributes map in format:
+     * [
+     *     attribute => param,
+     *     ...
+     * ].
```

Comment:
We tend to add `@return` attribute to all functions returning `array` type. So here you could add `@return string[]`